Dockerfile

```diff
@@ -1,43 +1,39 @@
 FROM alpine:3.6
 
-LABEL maintainer="Shopsys"
-
 ENV DOCKER_CLIENT_VERSION="18.05.0-ce"
 ENV TERRAFORM_VERSION="0.11.7"
 ENV KUBECTL_VERSION="1.11.1"
 ENV YQ_VERSION="2.1.1"
 ENV CLOUD_SDK_VERSION="206.0.0"
 ENV KUSTOMIZE_VERSION="1.0.10"
-ENV PATH /tmp/google-cloud-sdk/bin:$PATH
+ENV PATH /usr/bin/google-cloud-sdk/bin:$PATH
```

Comment:
I've talked about standard Linux path for installation such components with a colleague and it seems that `/usr/bin` should be used just for executables. If the component has it's own directory structure, it should be put into `/opt` and its `bin` directory should be added to paths, i.e. this line should be `ENV PATH /opt/google-cloud-sdk/bin:$PATH`

Reply:
similarly, user-installed binary files should be put into `/usr/local/bin/` if it exists. (And as you install kustomize there, we know it exists.)